src/plugins/googledrive/server/services/operation-service.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+const path = require('path');
+const process = require('process');
+const { google } = require('googleapis');
+module.exports = ({ strapi }) => ({
+  get: async (ctx) => {
+    const SERVICE_ACCOUNT_PATH = path.join(
+      process.cwd(),
+      'config',
+      'env',
+      'google_drive_service_account.json'
+    );
+    const SCOPES = ['https://www.googleapis.com/auth/drive.metadata.readonly'];
+    const auth = new google.auth.GoogleAuth({
+      keyFile: SERVICE_ACCOUNT_PATH,
+      scopes: SCOPES,
+    });
+    const drive = google.drive({ version: 'v3', auth });
+
+    if (ctx.request.query.fileId === undefined) {
```

Comment:
To address this https://github.com/dev-launchers/strapiv4/pull/250#discussion_r1881346470, we will need to get the file ID from `ctx.request.params` https://docs-v4.strapi.io/dev-docs/backend-customization/requests-responses#ctxrequest